### DISCUSSION TITLE
Customize Server Url and start page url

### DISCUSSION THREAD

**Discussion by woola168** [elsa-workflows/elsa-core#2830] [Q&A]
I have changed @page and add BasePath attribute on elsa-studio-root in _Host.cshtml like #1132 
and moved _Host.cshtml to Pages/ElsaCore/

But I still cant route to localhost:5000/ElsaCore 

I need to first go to localhost:5000 then press workflow definition to the right route
http://localhost:5000/ElsaCore/workflow-definitions
If I direct to http://localhost:5000/ElsaCore/ it will get me 404

plan to make the elsa server on our company server but just to add a route after origin url like https://qa.3rdchannel.com/ElsaCore
but didnt work 

UPDATE：
figure out that the question is can't load staticfile path , so how to change the path to load static file 
I have try to add staticFileOptions RequestPath but didint work

UPDATE 2：
Figure out that adding basepath at Host.cshtml can work and adding custom path after serverUrl

**Comment by sfmskywalker** [comment]
Hi @woola168, thanks for the updates. Does this mean you solved the routing issues?

**Reply by woola168** [reply]
yes , thanks for the reply 
solutions are from #1132 if anyone have this problem
